infrastructure/ansible/roles/database/flyway/defaults/main.yml

```diff
@@ -16,7 +16,7 @@ flyway_args:
     password: "{{ maps_db_password }}"
     url: "{{ flyway_db_url }}/{{ maps_db_name }}"
     migration:
-      zip: map_migrations.zip
+      zip: maps_database__migrations.zip
```

Comment:
Double underscore? I'm pretty sure this isn't intentional.

Reply:
Indeed also on line 20 the name was not updated either: map_migrations.zip

Both fixed in: d594b933a